documentation/java.asciidoc

```diff
@@ -22,22 +22,22 @@ Examples for create a new devon4j application:
 ```
 devon java create com.example.domain.myapp
 ```
-Will create an app with package `com.example.domain.myapp`, groupId `com.example.domain`, artifactId `myapp`, version `1.0.0-SNAPSHOT`, and h2 database.
+Will create an app with package `com.example.domain.myapp`, groupId `com.example.domain`, `artifactId` `myapp`, version `1.0.0-SNAPSHOT`, and h2 database.
```

Comment:
groupId not getting detected as spell mistake here?

Reply:
groupId  is in the dictionary